docs/hugo/content/introduction/authentication.md

```diff
@@ -43,6 +45,49 @@ stringData:
 EOF
 ```
 
+## Service Principal using Client Certificate
+
+### Prerequisites
+1. An existing Azure Service Principal.
+2. Certificate in ASCII format such as PEM, CER, or DER.
+
+To use Service Principal authentication via client certificate, specify an `aso-controller-settings` secret with the `AZURE_CLIENT_ID`, `AZURE_CLIENT_CERTIFICATE` and `AZURE_CLIENT_CERTIFICATE_PASSWORD`(optional) keys set.
+
+For more information about Service Principals, see [creating an Azure Service Principal using the Azure CLI](https://learn.microsoft.com/cli/azure/create-an-azure-service-principal-azure-cli#certificate-based-authentication).
+The `AZURE_CLIENT_ID` is sometimes also called the App ID. The `AZURE_CLIENT_CERTIFICATE` is the "certificate" returned by the command in the previously linked documentation.
+
+Use the following Bash script to set the environment variables for the `aso-controller-settings` secret:
+```bash
+export AZURE_CLIENT_ID="00000000-0000-0000-0000-00000000000"          # The client ID (sometimes called App Id) of the Service Principal.
+export AZURE_SUBSCRIPTION_ID="00000000-0000-0000-0000-00000000000"    # The Azure Subscription ID the identity is in.
+export AZURE_TENANT_ID="00000000-0000-0000-0000-00000000000"          # The Azure AAD Tenant the identity/subscription is associated with.
+export AZURE_CLIENT_CERTIFICATE="-----BEGIN PRIVATE KEY-----
```

Comment:
minor: does this export command actually work? Not sure how multiline exports work.

Reply:
yes, it does.. will have to do something like `export AZURE_CLIENT_CERTIFICATE=`cat certFile.pem`. Also, we have a livetest for this.